CMakeLists.txt

```diff
@@ -67,6 +73,43 @@ add_compile_options(-Wall -Wextra -W -Wshadow -Wformat -fexceptions)
 message(STATUS "Requested multimedia engine: ${USE_MULTIMEDIA}")
 message(STATUS "Requested build mode: ${CMAKE_BUILD_TYPE}")
 
+if(APPLE)
+  macro(_get_brew_prefix BREW_RECIPE_NAME VAR)
+    execute_process(
+      COMMAND "${BREW}" --prefix ${BREW_RECIPE_NAME}
+      OUTPUT_VARIABLE MAC_${VAR}_PREFIX
+      RESULT_VARIABLE MAC_${VAR}_RESULT
+    )
+
+    string(STRIP "${MAC_${VAR}_PREFIX}" MAC_${VAR}_PREFIX)
+  endmacro()
+
+  # ncurses version shipped with the latest macOS SDK is a fossilware version of 5.7 (A.D. 2008)
+  # We need 6.1, we need to take it from brew (for instance) and we need to be
```

Comment:
oh shit, i just say we depend on ncurses 6.1 because that's the earliest version i've tested with. we probably work on older versions just fine (maybe not?). did you try, and it failed? or was this done because you saw the 6.1 requirement?

Reply:
It was because of this requirement, yep. However, newer ncurses means up-to-date terminfo as well - so we do want to use the latest possible. However, I wonder if we shouldn't depend on [unibilium](https://github.com/neovim/unibilium) instead, what do you think?

Reply:
hrmm. depending on ncurses 6.1 makes us unusable on centos 7, debian jessie, ubuntu xenial, and derivatives. those are all pretty old, though i definitely see centos 7 all over the place.

5.7 is from 2008? christ almighty. i guess everyone on os x is using ITerm2 and it ships its own terminfo database?

you tell me. how much of an annoyance is this for some random mac person? i'm guessing not much of one? if it's no more difficult than what they're already doing to build notcurses, then sure, 6.1 is good.

unibilium looks good but not exactly compelling. what's the advantage over terminfo? i can't say i've run into any annoyances due to terminfo, and if terminfo gets new capabilities, unibilium has to catch up with it. i don't see anything really added by unibilium? it is available everywhere we're packaged, though, so that's not a problem.